### DISCUSSION TITLE
Docker compatability

### DISCUSSION THREAD

**Discussion by raindropworks** [UnchartedBull/OctoDash#3192] [Q&A] [answered]
Something tells me this will end up being a non-starter, but I'm still hopeful that I'm wrong.

I'm currently running Octoprint through a docker container on my home server, one of the reasons being the strong hardware, but also because I can integrate it in my web server to give it a proper website address instead of ip:port

Anyway, I came across a build that just happened to include a Prusa with an OctoDash display, and it's something that I'm very interested in. Something tells me though that this would only work with OctoPrint on a device with HDMI output and directly controlling the printer at the same time (something the server doesn't seem capable of doing.

Basically I think what I'm 'best case' trying to ask is if there's some way for a Pi to connect to an OctoPrint instance on another machine, and act as a bridge for OctoDash to work?

**Comment by queengooborg** [comment, marked as answer]
OctoDash can be installed on a separate device than the one that OctoPrint is running off of.  All you need to do is specify the OctoPrint server's URL when you first launch OctoDash (if it doesn't auto-detect)!
